server.js

```diff
@@ -24,7 +14,33 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('empty placeholder')
+})
+
+app.get('/avocado', (req, res) => { 
+  const { id } = req.query
+  const { maxPrice } = req.query
+  const { minPrice } = req.query
+  const { amount } = req.query
+
+  let filteredAvocado = avocadoSalesData
+
+  if(maxPrice){
+    filteredAvocado = filteredAvocado.filter((item) => item.averagePrice <= +maxPrice)
+  }
+
+  if(minPrice){
+    filteredAvocado = filteredAvocado.filter((item) => item.averagePrice >= +minPrice)
+  }
+
+  if(amount) {
+    filteredAvocado = filteredAvocado.slice(0,amount)
+  }
+
+  if(id) {
+    filteredAvocado = filteredAvocado.filter((item) => item.id === +id)
+  }
```

Comment:
This needs to be a separate endpoint `/avocados/:id` to meet blue requirements 🤓